docs/experiment/ppl/cmd/top.rst

```diff
@@ -0,0 +1,74 @@
+=============
+top
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Using ``top`` command to find the most frequent tuple of values of all fields in the field list.
+
+
+Syntax
+============
+top [N] <field-list> [by-clause]
+
+* N: number of results to return. **Default**: 10
```

Comment:
why the rare command doesn't has thsi paramater?

Reply:
Rare doesn't have N because it's not in the syntax given in this documentation: https://docs.splunk.com/Documentation/Splunk/8.0.2/SearchReference/Rare

Reply:
`limit` controls the number of results for rare but support for `top-options` isn't covered in this PR. 
Should I add parameter N for rare?

Reply:
no need now, just confirm,